### DISCUSSION TITLE
How to search Int filed

### DISCUSSION THREAD

**Discussion by sumeet030990** [prisma/prisma#14413] [Q&A]
schem.prisma
```
model dist_centers {
  id                     Int             @id @default(autoincrement())
  name                   String?         @db.VarChar(200)
  zip_code               String?         @db.VarChar(20)
  dc_id                  String?         @db.VarChar(8)
  shipping_day           Int?
  region_id              Int?
  enabled                Boolean?        @default(true)
  minimum_order_quantity Int?            @default(25)
  d365_id                String?         @db.VarChar(30)
  closing_note           String?         @db.VarChar(200)
  stores                 stores[]
  master_orders          master_orders[]
  orders                 orders[]
}
```

minimum_order_quantity contains value like 123,25,500 etc
so what I want to achieve is if I search 5 I should get result 25, 500
if I search 12 I should get 123 like that

for int type, I tried search, contains, startwith and all, but none of them are working 
and I am getting error
```
Unknown arg `contains` in where.minimum_order_quantity.contains for type IntNullableFilter. Did you mean `notIn`? Available args:\ntype IntNullableFilter {\n  equals?: Int | Null\n  in?: List<Int> | Null\n  notIn?: List<Int> | Null\n  lt?: Int\n  lte?: Int\n  gt?: Int\n  gte?: Int\n  not?: Int | NestedIntNullableFilter | Null\n}\n\n"
```

**Comment by austincrim** [comment]
Hey @sumeet030990 👋,

It is not possible to search for `Int` fields this way, you would have to use a string field if you want to query in this way.

You could potentially try to use `queryRaw` and cast `minimum_order_quantity` to a string in your query.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
